Remove usage of easy and some simple (#7110)
cakephp/docs: en/contributing/cakephp-coding-conventions.rst

        ...
    }

> This helps to keep the code flow more linear improving readability.

Reviewer: ```suggestion
This helps to keep the logic sequential which improves readability.
```

Careful when using empty()/isset()
==================================

> While ``empty()`` can seem like the correct function to use, it can mask errors

Reviewer: ```suggestion
While ``empty()`` often seems correct to use, it can mask errors
```
